### ISSUE TITLE
[A11y][ScreenReader]Compatible Framework tab does not convery compatible vs computed additional frameworks

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9421]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1746058

Environment Details:
App name: NuGet Service
#URL: https://dev.nugettest.org/

OS and Browser Details:
Edition Windows 11 Enterprise
OS build 22621.1105.
Chrome version:  109.0.5414.120 (Official Build) (64-bit)
Screen Reader: JAWS.

Repro Steps:
Hit the URL: https://dev.nugettest.org/
"NuGet" screen will open.
Tab to search button and press enter.
"Packages" screen will open then tab to any package like "Newtonsoft.Json " and select it by pressing enter key.
"Newtonsoft.Json " package screen will open.
Tab to "Frameworks" tab and press enter.
Observe whether screen reader is not announcing as a Compatible target framework(s) for dark blue color version.

Actual Result:
Screen reader is not announcing category of framework for version which are differentiated with color. For ex. Compatible target framework(s) for dark blue color version.

Expected Result:
Screen reader should announce category of framework for version which are differentiated with color.

Notes:
The screen reader on this is already fairly verbose. I am unsure how we would do this without making even more verbose. It may be ok to increase the verbosity, but it doesn't feel like a good solution to me.
